### ISSUE TITLE
Cannot add new friend/WebID in Friends pane

### ISSUE THREAD

**Issue by janschill** [solid/solid-ui#380]
I cannot add a new friend/WebID to my friends list in the Friends pane. The attached video shows how to reproduce the issue.

Safari: Version 14.0.3 (16610.4.3.1.4)
NSS: solid-server/5.6.4

```
Dropped URI list (2): https://michielbdejong.solidcommunity.net/profile/card#me
TypeError: e.forEach is not a function. (In 'e.forEach((function(e){var r=(0,a.sym)(e);c.log("Dropped on attachemnt "+e),n.push((0,a.st)(t,x,r,m))}))', 'e.forEach' is undefined)
```

Error is thrown here: https://github.com/search?q=org%3Asolid+Dropped+on+attachemnt&type=code

https://user-images.githubusercontent.com/26876570/109012447-50fdd200-76b2-11eb-8895-2427652aac57.mov

**Comment by janschill** [comment]
I tried with another session and it seems to work now. I cannot say what changed. Please close or leave open – your decision.

**Comment by janschill** [comment]
I've tried again. In Safari I get the issue described above. In Chrome it works as expected.

**Comment by angelo-v** [comment]
Does it depend on whether the friend list is initially empty?

**Comment by janschill** [comment]
I don't think so. My friend list is not empty and if I try to add you using Safari I get the same error as above:

<img width="1187" alt="image" src="https://user-images.githubusercontent.com/26876570/109425567-25982180-79e9-11eb-98c5-b65914c47874.png">

**Comment by angelo-v** [comment]
might be related to https://stackoverflow.com/questions/50299337/foreach-on-ios-safari-wont-work

**Comment by angelo-v** [comment]
Hmm, the forEach at this location should be fine. Do you have other issues with drag and drop in SolidOS on Safari or only on the friend list?

**Comment by janschill** [comment]
Same thing on the “Edit your profile” page.

https://user-images.githubusercontent.com/26876570/110004387-acb20600-7d17-11eb-90e1-81ed7bd519f0.mov

**Comment by theRealImy** [comment]
I could reproduce the problem locally on Safari and it shows 
![image](https://user-images.githubusercontent.com/4144203/130474646-f547ba1c-94d8-4638-9bfb-d12dcebd1b33.png)
